src/Microsoft.ML.Data/Prediction/CalibratorCatalog.cs

```diff
@@ -0,0 +1,465 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Core.Data;
+using Microsoft.ML.Calibrator;
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.Internal.Calibration;
+using Microsoft.ML.Runtime.Model;
+using Microsoft.ML.Runtime.Training;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.ML.Data;
```

Comment:
Do you even sort bro?

Reply:
I only sort in interviews :P

---
In reply to: [241130247](https://github.com/dotnet/machinelearning/pull/1816#discussion_r241130247) [](ancestors = 241130247)